R2API/ItemDrop/InteractableCalculator.cs

```diff
@@ -13,24 +13,32 @@ public class InteractableCalculator {
             "ShrineCleanse"
         };
         public static readonly Dictionary<string, ItemTier> TierConversion = new Dictionary<string, ItemTier> {
-            { "tier1", ItemTier.Tier1 },
-            { "tier2", ItemTier.Tier2 },
-            { "tier3", ItemTier.Tier3 },
-            { "boss", ItemTier.Boss },
-            { "lunar", ItemTier.Lunar }
+            { "tier1Tier", ItemTier.Tier1 },
```

Comment:
We should probably enumerate these strings

Reply:
I'm ok with that.